### ISSUE TITLE
Add label to form control in search bar

### ISSUE THREAD

**Issue by kburk1997** [YACS-RCOS/yacs-web#84]
In the search box, the `<input>` does not have a corresponding label.
From WAVE:
>An <input> (except types of image, submit, reset, button, or hidden), <select>, or <textarea> does not have a properly associated label text. A properly associated label is:
>a <label> element with a for attribute value that is equal to the id of a unique form control
>a <label> element that surrounds the form control, does not surround any other form controls, and does not reference another element with its for attribute
>a non-empty title attribute, or
>a non-empty aria-labelledby attribute.

**Comment by kburk1997** [comment]
Sorry about the ugly blockquote, github markdown does not want to work with me today

**Comment by sajeetharan** [comment]
@kburk1997  you mean add a name?
